src/chocolatey/infrastructure.app/nuget/NugetList.cs

```diff
@@ -26,12 +31,23 @@ public static class NugetList
     {
         public static IEnumerable<IPackage> GetPackages(ChocolateyConfiguration configuration, ILogger nugetLogger)
         {
+            return SearchPackagesImpl(configuration, nugetLogger);
+        }
+
+
+        public static int GetCount(ChocolateyConfiguration configuration, ILogger nugetLogger)
+        {
+            return SearchPackagesImpl(configuration, nugetLogger).Count();
+        }
+
+        private static IQueryable<IPackage> SearchPackagesImpl(ChocolateyConfiguration configuration, ILogger nugetLogger)

```

Comment:
Never `Impl`. It adds zero value to the method in my opinon. Resharper didn't call you out on the naming b/c of where you were for this method. 

My preference is something like `search_source_for_packages` or something like that. Naming is hard. It's easy to see when it is wrong, not always easy to call out what it should look like. ;)